lib/opencensus/trace/config.rb

```diff
@@ -64,6 +73,14 @@ class << self
       #     an object with a call method that takes a single options hash.
       #     See OpenCensus::Trace::Samplers. The initial value is a Probability
       #     sampler with a default rate.
+      # *   `exporter` The exporter to use. Must be an exporter, an object with
+      #     an export method that takes an array of Span objects. See
+      #     OpenCensus::Trace::Exporters. The initial value is a Logger exporter
+      #     that logs to STDOUT.
+      # *   `formatter` The trace context propagation formatter to use. Must be
+      #     a formatter, an object with serialize, deserialize, header_name,
```

Comment:
Wrap names in backticks to prevent markdown from italicizing stuff due to the underscores.